examples/test-umbrella/Readme.md

```diff
@@ -28,7 +28,7 @@ Start the topology on Mininet.
 $ cd endeavour && python examples/test-umbrella/mininet/simple_sdx.py
 ```
     
-In another terminal session.
+In another terminal session. Before run ryu-manager, correct the path in umbrella/fabric.cfg where the cloned endeavour directory is placed.
```

Comment:
This new sentence is not very clear.
Rather, the next command should include `cd endeavour &&`